src/wp-includes/class-wp-block-supports.php

```diff
@@ -181,7 +181,7 @@ function get_block_wrapper_attributes( $extra_attributes = array() ) {
 
 	// This is hardcoded on purpose.
 	// We only support a fixed list of attributes.
-	$attributes_to_merge = array( 'style', 'class', 'id' );
+	$attributes_to_merge = array( 'style', 'class', 'id', 'aria-label' );
```

Comment:
This PR also allows the `aria-label` attribute _by default_ in the `get_block_wrapper_attributes` function, otherwise blocks that have this support would need to explicitly specify the additional attribute, like so:

```php
$wrapper_attributes = get_block_wrapper_attributes( array( 'aria-label' => $attributes['ariaLabel'] ) );
```